packages/chakra-ui/src/range-field/RangeField.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useCallback } from "react";
+import { RangeField } from "./RangeField";
+import { fieldAtom, formAtom, useForm } from "form-atoms";
+import { StoryForm } from "../stories";
+import { zodValidate } from "form-atoms/zod";
+import { z } from "zod";
+import { NumberFieldAtom } from "@react-last-field/field/src";
+
+export default {
+  title: "RangeField",
+  component: RangeField,
+};
+
+const form = formAtom({
+  rating: fieldAtom({
```

Comment:
You can import new helper `numberField` which creates typed field atom with numeric validator & initial value
```suggestion
  rating: numberField()
```

Reply:
that will fix the typing!

Reply:
Thanks, should be solved: [7d1c68d](https://github.com/MiroslavPetrik/form-atoms-field/pull/8/commits/7d1c68d1cf89dfe132debdf2960c4030647ff626)